server/mongodb/index.ts

```diff
@@ -1,6 +1,6 @@
 import mongoose from "mongoose";
 
-export default async (): Promise<void> => {
+export async function connectToDB(): Promise<void> {
```

Comment:
I made this a named export because then we can export all models from this index.ts without having an additional subdirectory called "models"